README.md

```diff
@@ -8,9 +8,9 @@ You may need to create a "/var/lock" directory on OSX to fix a bug in the serial
 library. To do this open the Terminal application and run the following two
 commands:
 <br />
-   sudo mkdir /var/lock
+   '''sudo mkdir /var/lock'''
 <br />
-   sudo chmod 777 /var/lock
+   '''sudo chmod 777 /var/lock'''
```

Comment:
I think you used the wrong quote mark, maybe something like this would work?
> \`\`\`
>    sudo mkdir /var/lock
>   sudo chmod 777 /var/lock
> \`\`\`

You can see the preview in the "Files changed" tab by clicking on the paper icon.